js/script.js

```diff
@@ -21,7 +21,7 @@
   accordionVanillaJS();
 
   var accordionJQuery = function() {
-    $("#accordion-jquery h3").click(function() {
+    $("#accordion-jquery h3").on(function() {
```

Comment:
this isn't working - please take a look to jQuery documentation, how to use method `on`